sample/src/main/kotlin/app/futured/donutsample/ui/playground/PlaygroundActivity.kt

```diff
@@ -1,11 +1,13 @@
 package app.futured.donutsample.ui.playground
 
+import android.animation.ValueAnimator
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/thefuntasty/donut/blob/3317dfecd010d9a5776f0a2a2e33736acb814042/sample/src/main/kotlin/app/futured/donutsample/ui/playground/PlaygroundActivity.kt#L3"></span>Imports must be ordered in lexicographic order without any empty lines in-between</td>
    </tr>
  </tbody>
</table>